servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java

```diff
@@ -35,8 +35,10 @@ public static Allocator create(AllocatorType allocatorType, BlockMetadataManager
         return new GreedyAllocator(metaManager);
       case MAX_FREE:
         return new MaxFreeAllocator(metaManager);
+      case ROUND_ROBIN:
+        return new RoundRobinAllocator(metaManager);
       default:
-        return new GreedyAllocator(metaManager);
+        return new MaxFreeAllocator(metaManager);
```

Comment:
I changed it back now (committed).

~t

On 8 July 2015 at 13:38, Calvin Jia <notifications@github.com> wrote:

> In
> servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java
> <https://github.com/amplab/tachyon/pull/1040#discussion_r34195503>:
>
> >        default:
> > -        return new GreedyAllocator(metaManager);
> > +        return new MaxFreeAllocator(metaManager);
>
> Oh I see, let's keep the greedy version for now since thats the default in
> master.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amplab/tachyon/pull/1040/files#r34195503>.
>



-- 
Tianyin XU,
http://cseweb.ucsd.edu/~tixu/